WooCommerce/Classes/POS/Card Present Payments/CardPresentPaymentEvent.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+enum CardPresentPaymentEvent {
+    case idle
```

Comment:
Not sure about this one. In the context of the existing UI, it means "don't show anything", which would work fine, but it might be annoying in other contexts.

I considered adding a `completed` case, and we may still need it, but it feels a bit wrong. These events are published on a reusable stream, so the stream is never completed. Only the function call to collect payment or connect a reader is connected, and that's in its return value.

Reply:
We can revisit this case after integrating with the POS and see whether we still need this case or rename/split it.